examples/some-verifier/src/main.rs

```diff
@@ -189,6 +189,7 @@ async fn main() -> anyhow::Result<()> {
     let router = Router::new()
         .nest_service("/", serve_dir_service)
         .route("/verifications", post(add_verification))
+        .route("/verifications/remove", post(remove_verification))
```

Comment:
I agree, good suggestion!

Reply:
On second thought, I think it makes more sense to just use the same `Request` as when adding verifications, as we basically need to do the same checking mechanisms. Using parameters from the request path would require us to either check that they are the same values as included in the proof or modify the proof before checking it (both of which I think are unnecessary).

I'll change the route to be `DELETE /verifications`.

Reply:
I think `PATCH /verifications` makes more sense, since `DELETE /verifications` seems to imply that we are deleting the entire collection of verifications, whereas `PATCH` signifies that we are modifying it (by removing an entry).